### DISCUSSION TITLE
URL CHANGE

### DISCUSSION THREAD

**Discussion by samathanaprabu** [community/community#45691] [New to GitHub]
https://swannetworks.github.io/swannetwoks.github.io/

showing double URL but i need single url what i do.

**Comment by jge162** [comment]
Because you have two different github.io repositories. on one account. Look at the link, has `works` and then `wok` I think that might be the issue

**Reply by jge162** [reply]
very nice website btw

**Reply by samathanaprabu** [reply]
thank you dear....  i fixed the error

**Reply by jge162** [reply]
what was it, can you post answer to help others

**Reply by jge162** [reply]
I see you removed the redundant .io
